examples/misc/raptor.py

```diff
@@ -69,7 +69,7 @@ def task_state_cb(task, state):
     session   = rp.Session()
     try:
         pd = rp.PilotDescription(cfg.pilot_descr)
-        pd.cores   = n_masters * (cpn / masters_pn)
+        pd.cores   = n_masters * (cpn / masters_pn) + 256
```

Comment:
@andre-merzky , I do not understand why we still use this formula. According to the `raptor.cfg` and this equation, I will end up with ~ 650 cores.